extensions/amp-story/0.1/test/validator-amp-story-animation.out

```diff
@@ -0,0 +1,84 @@
+FAIL
+|  <!--
+|    Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+|
+|    Licensed under the Apache License, Version 2.0 (the "License");
+|    you may not use this file except in compliance with the License.
+|    You may obtain a copy of the License at
+|
+|        http://www.apache.org/licenses/LICENSE-2.0
+|
+|    Unless required by applicable law or agreed to in writing, software
+|    distributed under the License is distributed on an "AS-IS" BASIS,
+|    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+|    See the License for the specific language governing permissions and
+|    limitations under the license.
+|  -->
+|  <!--
+|    Test Description:
+|    Tests for the amp-story-animation tag.
+|  -->
+|  <!DOCTYPE html>
+|  <html ⚡>
+|    <head>
+|      <meta charset="utf-8" />
+|      <title>amp-story-animation</title>
+|      <link rel="canonical" href="/" />
+|      <meta
+|        name="viewport"
+|        content="width=device-width,minimum-scale=1,initial-scale=1"
+|      />
+|      <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+|      <script async src="https://cdn.ampproject.org/v0.js"></script>
+|      <script
+|        async
+|        custom-element="amp-story"
+|        src="https://cdn.ampproject.org/v0/amp-story-1.0.js"
+|      ></script>
+|      <script
+|        async
+|        custom-element="amp-animation"
+|        src="https://cdn.ampproject.org/v0/amp-animation-0.1.js"
+|      ></script>
```

Comment:
It's not essential to require the `amp-animation` script, but the extension will be slightly quicker to load if we do.

Reply:
We're starting to think of ways to improve LCP for a cold load, so for now loading `amp-animation` once the `amp-story` code is parsed sound better.
We'll keep this in mind and test each scenario (with/without the `amp-animation` script) as we work on code splitting.

Reply:
Removing requirement. Please note that once validation comes into effect without requiring the extension, we won't be able to add that requirement in the future.